test/e2e/serial/tests/workload_overhead.go

```diff
@@ -274,6 +274,158 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload overhea
 				}
 
 			})
+
+			It("[test_id:47582][tier2] Pod pending when resources requested + pod overhead don't fit on the target node; NRT objects are not updated", func() {
+				var targetNodeName string
+				var targetNrtInitial *nrtv1alpha1.NodeResourceTopology
+				var targetNrtListInitial nrtv1alpha1.NodeResourceTopologyList
+				var err error
+
+				// even if it is not a hard rule, and even if there are a LOT of edge cases, a good starting point is usually
+				// in the ballpark of 5x the base load. We start like this
+				podResources := corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("6"),
+					corev1.ResourceMemory: resource.MustParse("6Gi"),
+				}
+
+				// to avoid issues with fractional resources being unaccounted atm, we round up to requests;
+				// for the test proper, as low as cpu=100m and mem=100Mi would have been sufficient.
+				minRes := corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("1"),
+					corev1.ResourceMemory: resource.MustParse("1Gi"),
+				}
+
+				// need a zone with resources for overhead, pod and a little bit more to avoid zone saturation
+				klog.Infof("kubernetes pod fixed overhead: %s", e2ereslist.ToString(rtClass.Overhead.PodFixed))
+
+				const requiredNUMAZones = 2
+				By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNUMAZones))
```

Comment:
We are already filtering at the `BeforeEach` (lines 97-107) aren't we?

Reply:
+1 
also it isn't necessary to have two numas, but no need to consider that atm because the ci environments supports at least one worker with 2 numas.

Reply:
Good catch, will fix now.